XmlValidator/spec_exercise.xml

```diff
@@ -278,10 +278,11 @@
   <entry>
     <xpath>//entry/response/query</xpath>
     <children>
-      <one-of>
+      <any-number>
        <reference>block-or-minor-environment-elements</reference>
        <reference>inline-elements</reference>
-      </one-of>
+       <element>radio</element>
+      </any-number>
```

Comment:
Changing from one-of to any-number might have unexpected side effects. Please confirm two things here: the validator tests properly cover this change and that the test script maybe gives a warning if multiple inline elements or block or minor reference elements occur. Please check with Kosma about making this change and if they want to know about it.